proposals/single-source-documentation.md

```diff
@@ -0,0 +1,79 @@
+# Single-source documentation 
+
+Authors: @johnnymatthews 
+
+Initial PR: TBD 
+
+## Purpose and impact 
+
+### Background and intent
+
+Readers can find all IPFS documentation on a single website or in a single GitHub repository.
+
+### Assumptions and hypotheses
+
+This project assumes that folks who want to learn about IPFS don't like hopping around blog posts, websites, and GitHub repos to find the information they need.
+
+### User workflow example
+
+1. The reader goes to docs.ipfs.io.
+1. They find the information need.
+1. They use that information to perform a task.
+    a. If the task failed, they return to docs.ipfs.io for more information.
+
+### Impact
+
+🔥🔥🔥
+
+Developers get upset when a project's documentation does not answer their questions. Having docs all over the place is an excellent way to make sure we don't answer user questions correctly. Therefore, having docs in one place is a straight-forward way to increase developers' chances of finding the information they need.
+
+### Leverage
+
+🎯 
+
+Moving all docs into a single repo doesn't help _our_ developers build IPFS too much. But it really, _really_ helps developers who want to use IPFS but don't personally know an IPFS developer.
+
+### Confidence
+
+I don't understand this section. I'm very confident that what I've written so far is true and that completing this project will make developers' lives easier.
+
+## Project definition
+
+### Brief plan of attack
+
+[There's a project available in the ipfs-docs GitHub repo](https://github.com/ipfs/ipfs-docs/projects/3). This project includes a one week discovery phase where the team bounces around the internet, finding IPFS documentation, and determining whether it's still relevant or needs updating.
```

Comment:
Cool.  Is there any technology adjustments we need to make?  @warpfork had been wondering if there is any consistent set of docs tooling we should use across the InterPlanetary stack.

Reply:
> consistent set of docs tooling

There isn't a central _documentation-tooling_ repo, but that would be incredibly useful to have. Both IPFS Docs and Filecoin Docs (and I believe ProtoSchool to some extent) use VuePress as a static site generator. Filecoin docs was forked from IPFS docs last year, and there have been some major changes to the Filecoin repo that haven't filtered back to IPFS docs unfortunately.

Having one central repo that all projects can use would be very handy. However, it's out-of-scope for this project.